fycd/index.html

```diff
@@ -52,13 +52,13 @@ <h1>
                     </li>
                     <li>
                         <div id="gaad-year-nav">
-                            <button class="next-year btn-secondary" disabled="disabled">
-                                <span class="sr-only">Next year</span>
-                            </button>
-                            <span class="year">2024</span>
                             <button class="prev-year btn-secondary">
                                 <span class="sr-only">Previous year</span>
                             </button>
+                            <span class="year">2024</span>
+                            <button class="next-year btn-secondary" disabled="disabled">
```

Comment:
You can use aria-label= to add accessible text to a button, no need for the hidden span